src/types.ts

```
@@ -0,0 +1,49 @@
+    adminGroupId: string;
+    configId: string;
+    setupAt: string;
+    setupBy: TelegramUser;
+}
+
+export type ForwardContext = MessageContext<Bot<Record<string, never>, DeriveDefinitions>> & {
+    api: Bot<Record<string, never>, DeriveDefinitions>['api'];
+    args: null | string;
+    db: DynamoDBService;
+    me: TelegramUser;
+    bot: Bot;
```

ESLint

[error] Expected "bot" to come before "me".

(perfectionist/sort-object-types)